examples/large_models/segment_anything_fast/install_segment_anything_fast.sh

```diff
@@ -4,6 +4,7 @@
 pip uninstall torchtext torchdata torch torchvision torchaudio -y
 
 # Install nightly PyTorch and torchvision from the specified index URL
+pip install chardet
```

Comment:
What is this for ?

Reply:
Included this dependency because one of the previous benchmark runs failed because of the absence of this module: https://github.com/pytorch/serve/actions/runs/7255062486/job/19764967013

Reply:
Was this happening locally? I didnt notice this. If not, can you please move this to the dependencies section of the benchmark run?

Reply:
I think I noticed it locally as well. I will include it in `benchmark_model_dependencies.sh` script which you suggested.